app/services/nav-links.js

```diff
@@ -10,29 +10,27 @@ export default Service.extend({
 
   setupNavLinks() {
     let navlinks = [{
-      type: 'link',
-      href: '/core',
+      route: 'issues',
+      model: 'core',
```

Comment:
Later, when we add filters based on GitHub organizations (plus a filter for RFCs), we may want to consider updating the variable names to be more descriptive. Perhaps from `model` to `groupName` or `category`.

Because the words route and model mean something special in Ember, I think the variable names `route` and `model` may confuse future contributors.